src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionActions.tag

```diff
@@ -96,5 +89,29 @@
             </ul>
         </div>
     </div>
-    <tif:feedbackSessionPublishButton publishButton="${actions.publishButton}" buttonType="btn-default btn-xs margin-bottom-7px" />
-</c:if>
+    <div title="<%= Const.Tooltips.FEEDBACK_SESSION_RESULTS %>"
+         data-toggle="tooltip"
+         data-placement="top"
+         style="display: inline-block; padding-right: 5px;">
+        <div class="btn-group margin-bottom-7px">
+            <a class="btn btn-default btn-xs btn-tm-actions session-view-for-test"
+               href="${actions.resultsLink}">
+                Results
+            </a>
+            <button type="button"
+                    class="btn btn-default btn-xs btn-tm-actions dropdown-toggle"
+                    data-toggle="dropdown"
+                    aria-expanded="false">
+                <span class="caret"></span>
+            </button>
+            <ul class="dropdown-menu" role="menu">
+                <li>
+                    <a class="session-view-for-test" href="${actions.resultsLink}">View Results</a>
+                </li>
+                <li>
+                  <tif:feedbackSessionPublishButton publishButton="${actions.publishButton}" showTooltip="false"/>
+                </li>
+            </ul>
+        </div>
+    </div>
+</c:if>
```

Comment:
Sorry! Just realised an issue with this `</c:if>`. This results in instructors not being able to view results for a private session. However, publish button should still be disabled when it is a private session. One solution could be to use `<c:choose>` on line 49 instead and render the "Results" button differently depending on whether or not it is a private session to avoid the problem having a dropdown with only one option.